### PR TITLE
[DRAFT] Don't merge

### PR DESCRIPTION
**Pull request by dastrong**
### What does this PR do? What is the motivation?

Testing backticks in page titles and quotes in short descriptions

### Merge instructions

- [ ] Please merge after reviewing

### CODE REVIEW

File: content/en/continuous_integration/static_analysis/rules/_index.md [hunk 1]
```diff
@@ -1,8 +1,27 @@
 ---
-title: Static Analysis Rules
+title: Static `Analysis` Rules
```
Comment (dastrong) [inline content/en/continuous_integration/static_analysis/rules/_index.md:2]:
```suggestion
title: Static Analysis Rules
```

File: content/en/continuous_integration/static_analysis/rules/_index.md [hunk 2]
```diff
@@ -3,6 +3,25 @@ title: Static Analysis Rules
 kind: documentation
 description: View rules for multiple languages for Static Analysis.
 is_beta: true
+test_data:
+  - link: "/continuous_integration/static_analysis/rules/tsx-react/tsx-no-target-blank"
+    tag: "tsx-no-target-blank"
+    text: "Prevent target="_blank" attribute without rel="noreferrer""
```
Comment (dastrong) [inline content/en/continuous_integration/static_analysis/rules/_index.md:9]:
```suggestion
    text: "Prevent target=\"_blank\" attribute without rel=\"noreferrer\"
```

File: content/en/continuous_integration/static_analysis/rules/_index.md [hunk 3]
```diff
@@ -3,6 +3,25 @@ title: Static Analysis Rules
...
+  - link: "/continuous_integration/static_analysis/rules/typescript-best-practices/for-direction"
+    tag: "for-direction"
+    text: "Check "for" loop is moving in the right direction"
```
Comment (dastrong) [inline content/en/continuous_integration/static_analysis/rules/_index.md:12]:
```suggestion
    text: "Check \"for\" loop is moving in the right direction"
```

**Comment by aliciascott** [comment]
Added WIP label, please remove this label when it's ready for Docs review, thanks!

**Pull request closed without merging**